### PR TITLE
Updating topics-on-dockershim-and-cri-compatible-runtimes page formatting

### PR DESCRIPTION
**Pull request by nate-double-u**
Followup to https://github.com/kubernetes/website/pull/31528, updating page formatting.

### CODE REVIEW

**Comment by nate-double-u** [comment]
Deploy preview: https://deploy-preview-31661--kubernetes-io-main-staging.netlify.app/docs/reference/node/topics-on-dockershim-and-cri-compatible-runtimes/

Sorry, should have caught the formatting earlier.

File: content/en/docs/reference/node/topics-on-dockershim-and-cri-compatible-runtimes.md [hunk 1]
```diff
@@ -3,12 +3,11 @@ title: External Articles on dockershim Removal and on Using CRI-compatible Runti
 content_type: reference
 weight: 20
 ---
-<!-- overview -->
```
Comment (sftim) [inline content/en/docs/reference/node/topics-on-dockershim-and-cri-compatible-runtimes.md:6]: This _is_ an overview; I'd leave this line in (same for `<!-- body -->`)

Comment (nate-double-u) [reply content/en/docs/reference/node/topics-on-dockershim-and-cri-compatible-runtimes.md:6]: Do we need to add newlines then?

Currently the formatting looks like this:
Screen Shot 2022-02-08 at 12 40 44 PM

Comment (nate-double-u) [reply content/en/docs/reference/node/topics-on-dockershim-and-cri-compatible-runtimes.md:6]: (current page: https://deploy-preview-31528--kubernetes-io-main-staging.netlify.app/docs/reference/node/topics-on-dockershim-and-cri-compatible-runtimes/)

**Comment by jihoon-seo** [comment]
Related PR: #31662

**Comment by nate-double-u** [comment]
Closing in favour of https://github.com/kubernetes/website/pull/31662, Tim has a better solution 🙂

/close

**Pull request closed without merging**